WEB_SERVICE/WEB/_HIGH/GRAPHQL_DIRECTIVE_OVERLOADING/meta.json

```diff
@@ -0,0 +1,25 @@
+{
+  "risk_rating": "high",
+  "title": "Directive Overloading in GraphQL API",
+  "short_description": "Exploiting a GraphQL server by overloading queries with excessive directives, potentially leading to Denial of Service attacks.",
+  "references": {
+    "Checkmarx": "https://checkmarx.com/blog/alias-and-directive-overloading-in-graphql/",
+    "Beagle Security": "https://beaglesecurity.com/blog/article/graphql-attacks-vulnerabilities.html",
+    "Sylhare": "https://sylhare.github.io/2024/04/19/Graphql-vulnerabilities.html"
+  },
+  "privacy_issue": false,
+  "security_issue": true,
+  "categories": {
+    "CWE_TOP_25": [
+      "CWE_400"
+    ],
+    "PCI_STANDARDS":[
+      "REQ_6_2",
+      "REQ_6_4",
+      "REQ_11_3"
+    ],
+    "OWASP_MASVS_L2": [
+      "MSTG_PLATFORM_2"
+    ]
+  }
```

Comment:
How about the other standards like the owasp web. I don't think this list is complete.

Reply:
add these please :     "ASVS_V13_4_1": [
      "V13_4_1"
    ],
    "SOC2_CONTROLS": [
      "CC_2_1",
      "CC_4_1",
      "CC_7_1",
      "CC_7_2",
      "CC_7_4",
      "CC_7_5",
      "CC_9_1"
    ]

Reply:
added thanks